docs/minio-client-complete-guide.md

```diff
@@ -142,6 +142,15 @@ mc config host add gcs  https://storage.googleapis.com BKIKJAA5BMMU2RHO6IBB V8f1
 
 NOTE: Google Cloud Storage only supports Legacy Signature Version 2, so you have to pick - S3v2
 
+### Specify AccessKey and SecretKey through environment variables
```

Comment:
Can the title be changed to something else, as we are also specifying the endpoint here apart from the access credentials?